src/BlackSlope.Api/Program.cs

```diff
@@ -1,14 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Threading.Tasks;
+﻿using System.Reflection;
 using BlackSlope.Api.Common.Extensions;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Logging;
 
 namespace BlackSlope.Api
 {

```

Comment:
As part of the migration we should use Generic Host instead of Web Host, Web Host is for backward compatibility and is being deprecated.